src/frontend/components/Profile.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { kernel, useProfile } from '../db'
+
+export default function Profile () {
+  const [name, setName] = useState('')
+  const [tagline, setTagline] = useState('')
+  const [age, setAge] = useState(44)
```

Comment:
Default to `18` as minimum required age to use this service.

Reply:
Get it & DONE